cmd/webhook.go

```diff
@@ -33,6 +36,8 @@ const (
 	admissionWebhookAnnotationStatusKey = "filer-injector-webhook.das-zone.statcan/status"
 )
 
+const requestConfigMapName = "share-requests"
```

Comment:
Can you please rename to "requesting-shares"

Reply:
was an error on my part, this is really using the existing-shares configmap and has been updated to reflect that